context/app/static/js/components/Detail/FileBrowserDUA/FileBrowserDUA.jsx

```diff
@@ -21,28 +27,36 @@ function FileBrowserDUA(props) {
     >
       <DialogContent>
         <DialogContentText id="alert-dialog-description">
-          <h1 id="alert-dialog-title">HuBMAP Data Usage</h1>
+          <h1 id="alert-dialog-title">
+            {'HuBMAP '} <ObliqueSpan>{`${title} Data`}</ObliqueSpan> {' Usage'}
+          </h1>
 
           <h2>Appropriate Use</h2>
-          <p>
-            Users of HuBMAP open-data or processed data agree not to use the requested datasets, either alone or in
-            concert with any other information, to identify or contact individual participants or family members from
-            whom data and/or samples were collected.
-          </p>
+          <p>{appropriateUse}</p>
 
           <h2>Acknowledgement</h2>
           <p>
             Investigators using HuBMAP data in publications or presentations are requested to cite The Human Body at
-            Cellular Resolution: the NIH Human BioMolecular Atlas Program (doi:10.1038/s41586-019-1629-x) and to include
-            an acknowledgement of HuBMAP. Suggested language for such an acknowledgment is: “The results here are in
-            whole or part based upon data generated by the NIH Human BioMolecular Atlas Program (HuBMAP):{' '}
-            <a href="https://hubmapconsortium.org">https://hubmapconsortium.org</a>.”
+            Cellular Resolution: the NIH Human BioMolecular Atlas Program (doi:
+            <LightBlueLink href="https://doi.org/10.1038/s41586-019-1629-x" target="_blank" rel="noopener noreferrer">
+              10.1038/s41586-019-1629-x
+            </LightBlueLink>
+            ) and to include an acknowledgement of HuBMAP. Suggested language for such an acknowledgment is: “The
+            results here are in whole or part based upon data generated by the NIH Human BioMolecular Atlas Program
+            (HuBMAP):{' '}
+            <LightBlueLink href="https://hubmapconsortium.org" target="_blank" rel="noopener noreferrer">
+              https://hubmapconsortium.org
+            </LightBlueLink>
+            .”
           </p>
 
           <h2>Data Sharing Policy</h2>
           <p>
             The HuBMAP Data Sharing Policy can be found at{' '}
-            <a href="https://hubmapconsortium.org/policies/">https://hubmapconsortium.org/policies/</a>.
+            <LightBlueLink href="https://hubmapconsortium.org/policies/" target="_blank" rel="noopener noreferrer">
```

Comment:
Late in the game to ask this, but what is wrong with plain `<a>`?

Reply:
Need to use MUI components if we want font styles from the palette by default.